timer.js

```diff
@@ -22,20 +22,18 @@ function getLeftTime() {
   
   let totalMinutes = 0;
   
-  const items = document.querySelectorAll('.item-link.udlite-custom-focus-visible') // each item
-
+  const items = document.querySelectorAll('.item-link.item-link--common--RP3fp.ud-custom-focus-visible') // each item
+  console.log(items.length + " Items were found.")
```

Comment:
Could you remove the console log here?

Reply:
no worries 😄 
I have removed the log.
Have a nice one

Reply:
Merged, thank you for contribution :)